### PR TITLE
refactor: OPTIC-987: convert styl to scss

### PR DESCRIPTION
**Pull request by carly-bartel**
convert styl to scss in LS web/libs/editor/src/components/

### CODE REVIEW

File: web/libs/editor/src/components/AnnotationTabs/AnnotationTabs.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
+.annotation-tabs {
```
Comment (yyassi-heartex) [inline web/libs/editor/src/components/AnnotationTabs/AnnotationTabs.scss:1]: web/libs/editor/src/components/AnnotationTabs/AnnotationTabs.styl doesnt appear to have been removed

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10479239888)
> Successfully merged:  8 files changed, 9 insertions(+), 8 deletions(-)

File: web/libs/editor/src/components/AnnotationsCarousel/AnnotationsCarousel.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  &__carosel {
+    flex: 1;
+    display: flex;
+    gap: 2px;
+    white-space: nowrap;
+    padding-right: 70px;
+    position: relative;
+    transform: translateX(calc(-1 * var(0px)));
```
Comment (bmartel) [inline web/libs/editor/src/components/AnnotationsCarousel/AnnotationsCarousel.scss]:
```suggestion
    transform: translateX(calc(-1 * var(--carousel-left)));
```

**Review by yyassi-heartex — APPROVED** [review]

File: web/libs/editor/src/components/Entities/RegionItem.scss
Comment (bmartel) [inline web/libs/editor/src/components/Entities/RegionItem.scss:1]: I believe there is a missing deletion of the corresponding stylus file `web/libs/editor/src/components/Entities/RegionItem.styl`

**Review by bmartel — CHANGES_REQUESTED** [review]
There appears to be a couple stylus files remaining to be removed

Screenshot 2024-08-21 at 9 06 48 AM

**Review by bmartel — APPROVED** [review]
Approving based on the removals of the 2 leftover stylus files in the editor components.

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10512070838)
> Successfully merged:  17 files changed, 207 insertions(+), 396 deletions(-)

**Review by MihajloHoma — APPROVED** [review]

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10529525669)
> Successfully merged:  8 files changed, 16 insertions(+), 72 deletions(-)

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10529620844)
> Successfully merged: Already up to date.

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10530118914)
> Successfully merged:  5 files changed, 2 insertions(+), 50 deletions(-)

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10531483596)
> Successfully merged:  6 files changed, 198 insertions(+), 91 deletions(-)

**Pull request merged**